frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -0,0 +1,129 @@
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { click } from '@console/shared/src/test-utils/utils';
+import { isNodeReady } from '@console/shared/src/selectors/node';
+import {
+  addCapacityLbl,
+  addCapacityBtn,
+  ocsOp,
+  storageClusterView,
+  storageClusterRow,
+} from '../views/add-capacity.view';
+
+const namespace = 'openshift-storage';
+const kind = 'storagecluster';
+const storageCluster = JSON.parse(
+  execSync(`kubectl get -o json -n ${namespace} ${kind}`).toString(),
+);
+
+let clusterJSON;
+let previousCnt;
+let previousPods;
+let updatedClusterJSON;
+let updatedCnt;
+let updatedPods;
+let clusterExist = true;
+
+describe('Check availability of cluster', () => {
```

Comment:
Ideally, `describe` block would have an `it` block or nested `describe`. Or, you could use `tests` that is basically a merge of `describe + it`.

If we have HEALTH_WARN, does it mean cluster is not available?

Also, be specific about which cluster we are referring to. We also have NooBaa and OCS StorageCluster.

Reply:
As mentioned, please specify ocs cluster here and for rest other areas.